source/Octopus.Versioning/Octopus/OctopusVersionMask.cs

```diff
@@ -188,7 +188,7 @@ public override string EvaluateFromMask(string separator = "")
                     if (!IsPresent || string.IsNullOrEmpty(Value))
                         return string.Empty;
 
-                    var identifiers = Value.Split('.', '-', '_');
+                    var identifiers = Value.Split('.');
```

Comment:
This, along with the changes in OctopusVersionMaskParser.cs below, revert back to a more strict interpretation of masks.